sycl/test-e2e/Adapters/enqueue-arg-order-image.cpp

```diff
@@ -1,5 +1,7 @@
 // REQUIRES: aspect-ext_intel_legacy_image
 // UNSUPPORTED: target-amd
+// UNSUPPORTED-INTENDED: Legacy images aren't supported on AMD, but also fail
+// to compile, bindless images should be used instead.
```

Comment:
Nit but I'd make it a full-stop (or semi-colon): `to compile. Bindless images should be used instead` or `to compile; bindless images should be used instead`.

They're separate but related clauses. Same goes with most other examples of "bindless images should be used instead" I see here.

Reply:
Fixed up the comments with a full stop :+1: